roles/openshift_examples/examples-sync.sh

```diff
@@ -5,8 +5,8 @@
 #
 # This script should be run from openshift-ansible/roles/openshift_examples
 
-XPAAS_VERSION=ose-v1.2.0-1
-ORIGIN_VERSION=v1.1
+XPAAS_VERSION=master
```

Comment:
The xpaas team strongly prefers we never allow anything other than a tagged version of their content into the installer. I think we should leave that particular one as is.